alpha/scripts/utils/ExportByEntryListToDifferentStorage.php

```diff
@@ -0,0 +1,162 @@
+
+		foreach ($assets as /** @var flavorAsset $asset * */ $asset)
+		{
+			KalturaLog::debug('Handling asset with id ' . $asset->getId());
+			if ($externalStorage->shouldExportFlavorAsset($asset))
+			{
+				$criteria = new Criteria(FileSyncPeer::DATABASE_NAME);
+				$criteria->add(FileSyncPeer::OBJECT_ID, $asset->getId(), Criteria::EQUAL);
+				$criteria->add(FileSyncPeer::OBJECT_TYPE, FileSyncObjectType::ASSET);
+				$criteria->add(FileSyncPeer::OBJECT_SUB_TYPE, flavorAsset::FILE_SYNC_FLAVOR_ASSET_SUB_TYPE_ASSET);
+				$criteria->add(FileSyncPeer::DELETED_ID, null, Criteria::ISNULL);
+				$criteria->add(FileSyncPeer::DC, array($sourceDc, $storageIdDest), Criteria::IN);
+				$criteria->add(FileSyncPeer::PARTNER_ID, $asset->getPartnerId(), Criteria::EQUAL);
+				$criteria->addDescendingOrderByColumn(FileSyncPeer::VERSION);
+				$criteria->addAscendingOrderByColumn(FileSyncPeer::DC);
+				$fileSyncs = FileSyncPeer::doSelect($criteria);
+
+				$remoteDcFileSyncFound = false;
+				$maxVersion = -1;
+				$fileSyncToHandle = null;
+				//Get the Max Version and Local fileSync to handle
+				foreach ($fileSyncs as /** @var FileSync $fileSync * */ $fileSync)
+				{
+					if ($fileSync->getDc() == $sourceDc && $fileSync->getFileSize() > 0 && $fileSync->getVersion() > $maxVersion)
+					{
+						$fileSyncToHandle = $fileSync;
+					}
+				}
+				if (!$fileSyncToHandle)
+				{
+					KalturaLog::debug('No filesync to handle for flavor asset ' . $asset->getId());
+					continue;
+				}
+
+				// look for a sibling with same version on the remote storage
+				foreach ($fileSyncs as /** @var FileSync $fileSync * */ $fileSync)
+				{
+					if ($fileSync->getDc() === $storageIdDest && $fileSync->getVersion() == $fileSyncToHandle->getVersion())
+					{
+						$remoteDcFileSyncFound = true;
+					}
+				}
+
+				if ($remoteDcFileSyncFound)
+				{
+					KalturaLog::debug("Found file sync in remote dc [$storageIdDest] for assetId " . $asset->getId() . " . skipping exporting asset");
+					continue;
+				}
+
+				if ($fileSyncToHandle->getStatus() != flavorAsset::FLAVOR_ASSET_STATUS_READY)
+				{
+					KalturaLog::debug("Filesync " . $fileSyncToHandle->getId() . " in not ready " . $fileSyncToHandle->getStatus() . " - skipping asset");
+					continue;
+				}
+				try
+				{
+					KalturaLog::debug("Handling filesync " . $fileSyncToHandle->getId());
+					$newfileSync = $fileSyncToHandle->cloneToAnotherStorage($storageIdDest);
+					$newfileSync->save();
+					KalturaLog::debug('New FileSync created ' . $newfileSync->getId());
+				}
+				catch (Exception $e)
+				{
+					KalturaLog::warning("Could not create newFileSync for fileSync [" . $fileSync->getId() . "]" . $e->getMessage());
+				}
+			}
+			else
+			{
+				KalturaLog::debug('Asset ' . $asset->getId() . ' should not be Exported to remote storge ' . $storageIdDest);
+			}
+		}
+
+		kMemoryManager::clearMemory();
+		$count++;
+		if ($count == 1000)
```

Comment:
Sync we might be running few thousands.
Lets add the count to the 'Sleeping ...'  log so we can tell about the progress.
if($count++%1000 == 0)
can do the trick